app/views/mail.scala.html

```diff
@@ -0,0 +1,48 @@
+<table class="mailtable" style="width: 100%;">
+    <tr>
+        <td>
+            <table class="contenttable" style="width: 90%;margin: auto;">
+                <tr>
+                    <td class="logobox" style="text-align: center;"><img class="logo" src="https://media.licdn.com/media/AAEAAQAAAAAAAAKKAAAAJGYyN2NhODQ4LTQ5MTYtNGEwYy05NjA0LTAwOTNiMDNjNmNhOQ.png"/>
+                        <br>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        &nbsp;
+                    </td>
+                </tr>
+                <tr>
+                    <td class="titlebox" style="background-color: rgba(0, 117, 162, 0.4);padding: 20px;">
+                        <span class="titleboxtitle" style="font-family: Arial;font-size: 20px;font-weight: bold;color: #000;">Friday Lunch !!!</span>
+                        <br><p></p>
+                        <span class="titleboxtext" style="font-family: Arial;font-size: 14px;color: #000;">It's a gentle reminder to to fill your availablity <a href="http://lunatech-lunch.cleverapps.io/" style="font-size: 18px;font-weight: bold;">Lunatech-lunch</a>.
+							<br>
+							Please fill your food preferences also. </span>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        &nbsp;
+                    </td>
+                </tr>
+                <tr>
+                    <td class="titlebox2" style="background-color: rgba(213, 86, 114, 0.4);padding: 20px;">
+                        <br><p></p>
+                        <span class="titleboxtext" style="font-family: Arial;font-size: 14px;color: #000;">
+                            <li>If you have any suggestions please mail us at developers@@lunatech.com</li>
+                            <li>If you find any issue in application please log the issue <a href="https://github.com/lunatech-labs/lunatech-lunch-planner/issues">here</a> or resolve it :)</li>
+
+						</span>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        &nbsp;
+                    </td>
+                </tr>
+            </table>
+        </td>
+    </tr>
+
+</table>
```

Comment:
New line at the enf of the file here as well - perhaps you should change your IDE settings .